registry/application/services/slack_chat_operation.py

```diff
@@ -455,6 +455,12 @@ def generate_view_service_modal(self, org_id, service, requested_at, comment):
     def generate_view_org_modal(self, org, requested_at, comment):
         org_id = "None" if not org.id else org.id
         organization_name = "None" if not org.name else org.name
+        duns_no = "None" if not org.duns_no else org.duns_no
```

Comment:
lets add test cases as well pls

Reply:
Have to see how to test the slack template? It will not be straight forward.